ansible/services.yml

```diff
@@ -15,7 +15,7 @@
     nodejs_version: '6.x'
 
     apt_packages:
-      pkg:
+      - pkg:
```

Comment:
this looks even more wrong to me 🤔 

shouldn't it be:

```
    apt_packages:
        - nodejs
        - git
        - make
        - unzip
```

? 

It's used in https://github.com/dincho/pagrant/pull/45/files#diff-77b559aed21060664f6c5e1d37b77b25L137 only and I wonder why it needs a multi-dimentional array ?

Reply:
https://github.com/dincho/pagrant/pull/43#discussion_r393073580